app/views/remote_follow/new.html.haml

```diff
@@ -7,7 +7,7 @@
   = simple_form_for @remote_follow, as: :remote_follow, url: account_remote_follow_path(@account) do |f|
     = render 'shared/error_messages', object: @remote_follow
 
-    = f.input :acct, placeholder: t('remote_follow.acct')
+    = f.input :acct, placeholder: t('remote_follow.acct'), value: @remote_follow.acct
```

Comment:
This is unnecessary as simple_form should pull the :acct value from @remote_follow, which you already set in the controller.. I think. Have you tested it? Otherwise this looks good to me